scholarly/publication_parser.py

```diff
@@ -58,7 +58,7 @@ def _load_url(self, url: str):
         # this is temporary until setup json file
         self._soup = self._nav._get_soup(url)
         self._pos = 0
-        self._rows = self._soup.find_all('div', class_='gs_r gs_or gs_scl') + self._soup.find_all('div', class_='gsc_mpat_ttl')
+        self._rows = self._soup.find_all('div', class_='gs_r gs_or gs_scl') + self._soup.find_all('div', class_='gs_r gs_or gs_scl gs_fmar') + self._soup.find_all('div', class_='gsc_mpat_ttl')
```

Comment:
Did you get these classes from the specific example you've tested here?

Reply:
Yes, but afterwards I tested multiple queries where the output of each was a single publication. Each of them has the same structure, a single publication appears on the google scholar page with a full abstract and the div element representing the publication contains these classes.